packages/pattern-commands/src/lib/structures/PatternCommand.ts

```diff
@@ -3,9 +3,11 @@ import type { Awaitable, Message } from 'discord.js';
 
 export abstract class PatternCommand extends Command<Args, PatternCommand.Options> {
 	public readonly chance: number;
+	public readonly isFullMatchName: boolean;
```

Comment:
```suggestion
	public readonly matchFullName: boolean;
```
or something similar maybe? `isFullMatchName` just doesn't seems right, maybe `isFullNameToBeMatched` or `isFullNameMatch`